test/index.js

```diff
@@ -190,3 +192,22 @@ test("works with no styles at all", function(t) {
 
   t.end()
 })
+
+test("works with caching", function(t) {
+  var opts = {path: importsDir, cacheDir: cacheDir}
+  var postcssOpts = {from: "test/fixtures/relative-to-source.css"}
+  var name = "relative-to-source"
+  var actual = postcss()
+    .use(atImport(opts))
+    .process(read("fixtures/" + name), postcssOpts)
+    .css.trim()
+  var expected = read("fixtures/" + name + ".expected")
+  t.equal(actual, expected, "put content in cache")
+  actual = postcss()
+    .use(atImport(opts))
+    .process(read("fixtures/" + name), postcssOpts)
+    .css.trim()
+  expected = read("fixtures/" + name + ".expected")
+  t.equal(actual, expected, "read content from cache")
+  t.end()
+})
```

Comment:
The test does not prove anything right now :/. We should probably add something to track the cache usage.